torchao/prototype/low_bit_optim/cpu_offload.py

```diff
@@ -38,51 +38,53 @@ def __init__(
         if not isinstance(param_groups[0], dict):
             param_groups = [{"params": param_groups}]
 
-        self.param_cuda2cpu_map = dict()
+        self.param_d2h_map = dict()
         self.optim_dict = dict()
-        self.stream = torch.cuda.Stream()
+        self.device = get_available_devices()[-1]
+        assert ("cuda" in self.device or "xpu" in self.device), "CPU Offload currently only supports CUDA & XPU"
```

Comment:
Shouldn't this be `self.device in ("cuda", "xpu")` instead?

Reply:
Changed according to ur comment.